src/_shared/components/CardActionButtonRow/CardActionButtonRow.tsx

```diff
@@ -32,14 +33,25 @@ interface CardActionButtonRowProps {
   /**
    * Callback for the "Reject" (trash) button
    */
-  onReject: VoidFunction;
+  onReject?: VoidFunction;
+
+  /**
+   * Callback for the "Remove" (X) button
+   */
+  onRemove?: VoidFunction;
 }
 
 export const CardActionButtonRow: React.FC<CardActionButtonRowProps> = (
   props,
 ): JSX.Element => {
-  const { onEdit, onUnschedule, onReschedule, onMoveToBottom, onReject } =
-    props;
+  const {
+    onEdit,
+    onUnschedule,
+    onReschedule,
+    onMoveToBottom,
+    onReject,
+    onRemove,
+  } = props;
```

Comment:
not needed yet, but a potential thought for the future - would it make sense to abstract the actions a bit more, so this component doesn't need to know about all available actions across all card implementations?

a consideration for if/when we expand the available card actions...

Reply:
yeah, i think that's a good ticket to file. thanks!